.github/workflows/rust.yml

```diff
@@ -28,11 +28,23 @@ jobs:
           command: fmt
           args: --all -- --check
 
-      - name: Run cargo clippy
+      - name: Run cargo clippy - flamegraph
```

Comment:
How can I forbid these redundant tests?

I cannot use `--all-features` because we cannot use `prost` and `rust-protobuf` at the same time.

Reply:
(Though none of these features contain tests :cry: